CMakeLists.txt

```diff
@@ -240,7 +240,7 @@ if(NOT WIN32 AND NOT EMSCRIPTEN)
         quickjs-libc.c
         run-test262.c
     )
-    target_compile_definitions(run-test262 PRIVATE ${qjs_defines})
+    target_compile_definitions(run-test262 PRIVATE ${qjs_defines} JS_RUN_TEST262)
```

Comment:
The problem is this flag won't be set for que qjs target, which is the one that needs it.

Reply:
Yes, this flag is needed by qjs when building shared library, but I'm not sure if spreading this flag is a good idea.

Reply:
IMHO it's not. Hiding that one symbol seems to be more trouble than it's worth.

Reply:
Yep, too many changes introduced. I'd close this.